homeassistant/helpers/config_validation.py

```diff
@@ -737,7 +704,7 @@ def extend(self, schema, required=None, extra=None):
         return HASchema(ret.schema, required=required, extra=self.extra)
 
 
-PLATFORM_SCHEMA = HASchema({
+PLATFORM_SCHEMA = vol.Schema({
```

Comment:
This will also remove the extra keys warning - is that also supposed to be part of this PR? Shouldn't we also remove the class too?

Reply:
The extra keys was supposed to be removed after 0.92, so think this would be a fitting moment? I didn't remove the class but you're right, I should .

Reply:
Yeah, we can remove it no problem.

Was just wondering because it was not part of the description/breaking change section and it looked like this slipped in.

Reply:
Oh good point, addressed it.